utils/client.py

```diff
@@ -23,7 +23,7 @@
     'donor': ['donor', (), (), (), ()],
     'donors': ['donors', (), (), (), ()],
     'event': ['events', ('', 'id'), ('name', 'description', 'start_date', 'end_date', 'team_limit',
-                                     'team_building_start', 'team_building_end'), ('id',),
+                                     'team_building_start', 'team_building_end', 'default_steps'), ('id',),
```

Comment:
This line is going past 100 chars, can you reindent?
```suggestion
                                     'team_building_start', 'team_building_end', 'default_steps'),
```